temporal-testing/src/main/java/io/temporal/testing/TestActivityEnvironment.java

```diff
@@ -78,6 +81,36 @@ static TestActivityEnvironment newInstance(TestEnvironmentOptions options) {
    */
   <T> T newActivityStub(Class<T> activityInterface);
 
+  /**
+   * Creates a stub that can be used to invoke activities registered through {@link
+   * #registerActivitiesImplementations(Object...)}.
+   *
+   * @param <T> Type of the activity interface.
+   * @param activityInterface activity interface class that the object under test implements
+   * @param options options that specify the activity invocation parameters
+   * @param activityMethodOptions activity method-specific invocation parameters
```

Comment:
Document that the key value is the activity type which is by default is the method name with the first capitalized letter.